backend/src/game_objects/cell.rs

```diff
@@ -0,0 +1,43 @@
+use super::unit::Unit;
+use serde::Serialize;
+
+#[derive(Clone, Serialize)]
+pub struct Cell {
+    pub x: u32,
+    pub y: u32,
+    pub unit: Option<Unit>,
```

Comment:
What about walls?

Reply:
added

Reply:
I think that is a possible approach.
But certainly not the only one.
What if there will be a number of obstacles, and maybe several possible graphic representations of a wall?

I don't think that "Wall" should be an innate property of the Cell, it looks more like the thing the Cell may contain.
So I think it will be more logical to abstract Units and Walls instead.

Reply:
Rename Cell -> Hex

Reply:
renamed

Reply:
i can't answer in one thread to both of you))

Reply:
ok, i made what i can, but i didn't achieve any pretty solution for walls

it seems i need an assistance from skillful rustacean like u guys ;)